sys/include/net/gnrc/gomach/gomach.h

```diff
@@ -7,10 +7,20 @@
  */
 
 /**
- * @defgroup    net_gnrc_gomach A traffic-adaptive multi-channel MAC
+ * @defgroup    net_gnrc_gomach GoMacH
  * @ingroup     net_gnrc
- * @brief       A traffic adaptive MAC protocol that provides high traffic
- *              adaptability, high energy efficiency and high robustness.
+ * @brief       GoMacH is, "a General, nearly Optimal MAC protocol
+ *              for multi-Hop communications", for IEEE 802.15.4 IoT/WSNs networks.
+ *              It is designed to be a traffic adaptive MAC protocol that provides
+ *              high traffic adaptability, high energy efficiency and high robustness.
+ *
+ * ## GoMacH's main features
+ * doesn't rely on global synchronization
```

Comment:
mh... now it isn't a list anymore I just realized (the stars are just from the C-comment). Probably most readable to do it with dashes:

```
 *  - doesn't rely on global synchronization
 *  - ...